packages/cmd/src/help/DefaultHelpGenerator.ts

```diff
@@ -18,6 +18,7 @@ import { ImperativeError } from "../../../error";
 import { IHelpGeneratorParms } from "./doc/IHelpGeneratorParms";
 import { IHelpGeneratorFactoryParms } from "./doc/IHelpGeneratorFactoryParms";
 import { compareCommands, ICommandDefinition } from "../../src/doc/ICommandDefinition";
+import { ansiRegex } from "ansi-colors";
```

Comment:
hi @t1m0thyj @zFernand0 - there was an overlook from my side and I'm getting an error upon installing @zowe/cli@zowe-v2-lts:

```
npm ERR! code 1
npm ERR! path C:\Users\so891058\AppData\Roaming\npm\node_modules\@zowe\cli
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c node ./scripts/validatePlugins && node ./scripts/printSuccessMessage
npm ERR! Since you re-installed Zowe CLI, we are re-validating any plugins.
npm ERR! node:internal/modules/cjs/loader:936
npm ERR!   throw err;
npm ERR!   ^
npm ERR!
npm ERR! Error: Cannot find module 'ansi-colors'
npm ERR! Require stack:
npm ERR! - C:\Users\so891058\AppData\Roaming\npm\node_modules\@zowe\cli\node_modules\@zowe\imperative\lib\cmd\src\help\DefaultHelpGenerator.js
```

It's listed in devDependencies only, should it be included in `dependencies` as well in `package.json` ?

Reply:
addressed in #849